routes/comments.js

```diff
@@ -4,5 +4,8 @@ const commentController = require("../controller/commentsController");
 
 router.use("/:commentId/replies", repliesRoutes);
 
+//delete comments
+router.delete("/comments/:commentId", commentController.deleteComment);
```

Comment:
You can omit the `/comments` path as it is already used for the comments routes. See the server.js file to see what I mean.